### PR TITLE
EN6-142 Fixing sort names to match frontend expectations

### PR DESCRIPTION
**Pull request by BrenoQVDS**
Normalized columns named received by the frontend to match the DB

### CODE REVIEW

**Comment by entando-service** [comment]
All is Well

File: plugins/entando-plugin-jacms/src/test/java/org/entando/entando/plugins/jacms/web/content/ContentControllerIntegrationTest.java [hunk 1]
```diff
@@ -857,6 +858,196 @@ public void testLoadOrderedPublicEvents_4() throws Throwable {
...
+        result.andExpect(status().isOk());
+        String[] expectedFreeContentsId = {"EVN25", "EVN21",
+                "EVN20", "EVN194", "EVN193", "EVN192", "EVN191", "EVN23", "EVN24"};
+        int payloadSize = JsonPath.read(bodyResult, "$.payload.size()");
+        Assert.assertEquals(expectedFreeContentsId.length, payloadSize);
+        for (int i = 0; i < expectedFreeContentsId.length; i++) {
+            String expectedId = expectedFreeContentsId[i];
+            String extractedId = JsonPath.read(bodyResult, "$.payload[" + i + "].id");
+            Assert.assertEquals(expectedId, extractedId);
+        }
```
Comment (ffleandro) [inline plugins/entando-plugin-jacms/src/test/java/org/entando/entando/plugins/jacms/web/content/ContentControllerIntegrationTest.java]: Check code style for building mvc test rules on my latest commit on PR #556

File: plugins/entando-plugin-jacms/src/test/java/org/entando/entando/plugins/jacms/web/content/ContentControllerIntegrationTest.java [hunk 2]
```diff
@@ -857,6 +858,196 @@ public void testLoadOrderedPublicEvents_4() throws Throwable {
...
+                        .param("filters[0].operator", "eq")
+                        .param("filters[0].value", "EVN")
+                        .sessionAttr("user", user)
+                        .header("Authorization", "Bearer " + accessToken));
+        String bodyResult = result.andReturn().getResponse().getContentAsString();
+        result.andExpect(status().isOk());
+        String expectedContentId = "EVN25";
+        int payloadSize = JsonPath.read(bodyResult, "$.payload.size()");
+        String extractedId = JsonPath.read(bodyResult, "$.payload[" + 0 + "].id");
+        Assert.assertEquals(expectedContentId, extractedId);
```
Comment (ffleandro) [inline plugins/entando-plugin-jacms/src/test/java/org/entando/entando/plugins/jacms/web/content/ContentControllerIntegrationTest.java]: Code style

File: plugins/entando-plugin-jacms/src/test/java/org/entando/entando/plugins/jacms/web/content/ContentControllerIntegrationTest.java [hunk 3]
```diff
@@ -857,6 +858,196 @@ public void testLoadOrderedPublicEvents_4() throws Throwable {
...
+        String bodyResult = result.andReturn().getResponse().getContentAsString();
+        result.andExpect(status().isOk());
+        String[] expectedFreeContentsId = {};
+        int payloadSize = JsonPath.read(bodyResult, "$.payload.size()");
+        Assert.assertEquals(expectedFreeContentsId.length, payloadSize);
+        for (int i = 0; i < expectedFreeContentsId.length; i++) {
+            String expectedId = expectedFreeContentsId[i];
+            String extractedId = JsonPath.read(bodyResult, "$.payload[" + i + "].id");
+            Assert.assertEquals(expectedId, extractedId);
+        }
```
Comment (ffleandro) [inline plugins/entando-plugin-jacms/src/test/java/org/entando/entando/plugins/jacms/web/content/ContentControllerIntegrationTest.java]: Code style

File: plugins/entando-plugin-jacms/src/test/java/org/entando/entando/plugins/jacms/web/content/ContentControllerIntegrationTest.java [hunk 4]
```diff
@@ -857,6 +858,196 @@ public void testLoadOrderedPublicEvents_4() throws Throwable {
...
+        result.andExpect(status().isOk());
+        String[] expectedFreeContentsId = {"EVN20", "EVN192",
+                "EVN23", "EVN24", "EVN21", "EVN25", "EVN191", "EVN194", "EVN193"};
+        int payloadSize = JsonPath.read(bodyResult, "$.payload.size()");
+        Assert.assertEquals(expectedFreeContentsId.length, payloadSize);
+        for (int i = 0; i < expectedFreeContentsId.length; i++) {
+            String expectedId = expectedFreeContentsId[i];
+            String extractedId = JsonPath.read(bodyResult, "$.payload[" + i + "].id");
+            Assert.assertEquals(expectedId, extractedId);
+        }
```
Comment (ffleandro) [inline plugins/entando-plugin-jacms/src/test/java/org/entando/entando/plugins/jacms/web/content/ContentControllerIntegrationTest.java]: Code style

File: plugins/entando-plugin-jacms/src/test/java/org/entando/entando/plugins/jacms/web/content/ContentControllerIntegrationTest.java [hunk 5]
```diff
@@ -857,6 +858,196 @@ public void testLoadOrderedPublicEvents_4() throws Throwable {
...
+        result.andExpect(status().isOk());
+        String[] expectedFreeContentsId = {"EVN193", "EVN194",
+                "EVN191", "EVN25", "EVN21", "EVN24", "EVN23", "EVN192", "EVN20"};
+        int payloadSize = JsonPath.read(bodyResult, "$.payload.size()");
+        Assert.assertEquals(expectedFreeContentsId.length, payloadSize);
+        for (int i = 0; i < expectedFreeContentsId.length; i++) {
+            String expectedId = expectedFreeContentsId[i];
+            String extractedId = JsonPath.read(bodyResult, "$.payload[" + i + "].id");
+            Assert.assertEquals(expectedId, extractedId);
+        }
```
Comment (ffleandro) [inline plugins/entando-plugin-jacms/src/test/java/org/entando/entando/plugins/jacms/web/content/ContentControllerIntegrationTest.java]: Code style

File: plugins/entando-plugin-jacms/src/test/java/org/entando/entando/plugins/jacms/web/content/ContentControllerIntegrationTest.java [hunk 6]
```diff
@@ -857,6 +858,196 @@ public void testLoadOrderedPublicEvents_4() throws Throwable {
...
+        result.andExpect(status().isOk());
+        String[] expectedFreeContentsId = {"EVN21", "EVN20",
+                "EVN25", "EVN24", "EVN23", "EVN192", "EVN191", "EVN194", "EVN193"};
+        int payloadSize = JsonPath.read(bodyResult, "$.payload.size()");
+        Assert.assertEquals(expectedFreeContentsId.length, payloadSize);
+        for (int i = 0; i < expectedFreeContentsId.length; i++) {
+            String expectedId = expectedFreeContentsId[i];
+            String extractedId = JsonPath.read(bodyResult, "$.payload[" + i + "].id");
+            Assert.assertEquals(expectedId, extractedId);
+        }
```
Comment (ffleandro) [inline plugins/entando-plugin-jacms/src/test/java/org/entando/entando/plugins/jacms/web/content/ContentControllerIntegrationTest.java]: Code style

File: plugins/entando-plugin-jacms/src/test/java/org/entando/entando/plugins/jacms/web/content/ContentControllerIntegrationTest.java [hunk 7]
```diff
@@ -857,6 +858,196 @@ public void testLoadOrderedPublicEvents_4() throws Throwable {
...
+        result.andExpect(status().isOk());
+        String[] expectedFreeContentsId = {"EVN21", "EVN25",
+                "EVN24", "EVN23", "EVN20", "EVN194", "EVN193", "EVN192", "EVN191"};
+        int payloadSize = JsonPath.read(bodyResult, "$.payload.size()");
+        Assert.assertEquals(expectedFreeContentsId.length, payloadSize);
+        for (int i = 0; i < expectedFreeContentsId.length; i++) {
+            String expectedId = expectedFreeContentsId[i];
+            String extractedId = JsonPath.read(bodyResult, "$.payload[" + i + "].id");
+            Assert.assertEquals(expectedId, extractedId);
+        }
```
Comment (ffleandro) [inline plugins/entando-plugin-jacms/src/test/java/org/entando/entando/plugins/jacms/web/content/ContentControllerIntegrationTest.java]: Code style

**Comment by entando-service** [comment]
All is Well

**Pull request merged**